release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -257,6 +257,14 @@ With this release, the etcd tuning parameters can be set to values that optimize
 * `Standard`
 * `Slower`
 
+[discrete]
+[id="egress-ip-etp-local-support"]
+=== Support for EgressIP with ExternalTrafficPolicy=Local services
+
+Previously, the use of `ExternalTrafficPolicy=Local` was unsupported with EgressIP configurations when using MetalLB or an F5 load balancer. When attempting this configuration, connections reaching pods via the OVN-Kubernetes routing gateway were returned through EgressIP if an EgressIP was configured for a namespace, which affects how responses to incoming connections are handled. This led  to service disruption when `externalTrafficPolicy` was set to `local`. Consequently, this cause multiple issues for customers. For MetalLB, this resulted in inaccessible load balancer IPs and broke the expected behavior of the service. For F5 LBs, it disrupted traffic routing and led to dropped connections and service unavailability.
+
+With {product-title} {product-version}, OVN-Kubernetes now supports the use of `ExternalTrafficPolicy=Local` services and `EgressIP` configurations at the same time on the same set of selected pods. OVN-Kubernetes now only reroutes the traffic originating from the pods towards the egress node while routing the responses to ingress service traffic via the same node where the pod lives.
```

Comment:
```suggestion
With {product-title} {product-version}, OVN-Kubernetes now supports the use of `ExternalTrafficPolicy=Local` services and `EgressIP` configurations at the same time on the same set of selected pods. OVN-Kubernetes now only reroutes the traffic originating from the EgressIP pods towards the egress node while routing the responses to ingress service traffic from the EgressIP pods via the same node where the pod lives.
```

Reply:
@tssurya yeah sorry--I had ran out of time yesterday and had to bail before getting a chance to fix it :)